docs/en/integrations/data-ingestion/data-formats/sql.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_label: SQL dumps
+sidebar_position: 6
+slug: /en/integrations/data-formats/sql
+---
+
+# Inserting and dumping SQL data in ClickHouse
+
+ClickHouse can be easily integrated into OLTP database infrastructures in many ways. One way is to transfer data between other databases and ClickHouse using SQL dumps.
+
+## Creating SQL dumps
+
+Data can be dumped in SQL format using [SQLInsert](/docs/en/interfaces/formats.md/#sqlinsert). ClickHouse will write data in `INSERT INTO <table name> VALUES(...` form and use [`output_format_sql_insert_table_name`](/docs/en/operations/settings/settings.md/#output_format_sql_insert_table_name) settings option as a table name:
+
+```sql
+SET output_format_sql_insert_table_name = 'some_table';
+SELECT * FROM some_data
+INTO OUTFILE 'dump.sql'
+FORMAT SQLInsert
+```
+
+Columns names can be omitted by disabling [`output_format_sql_insert_include_column_names`](/docs/en/operations/settings/settings.md/#output_format_sql_insert_include_column_names) option:
+
+```sql
+SET output_format_sql_insert_include_column_names = 0
+```
+
+Now we can feed [dump.sql](assets/dump.sql) file to another OLTP database:
+
+```bash
+mysql some_db < dump.sql
+```
+
+We assume that the `some_table` table exists in the `some_db` MySQL database.
```

Comment:
You can control the table name @mrcrypster - I nearly always need to do this. There is a setting we should mention.

Also, often tools such as psql require smaller batches - or it puts all the values in a single batch. There is a setting for this  -1000 is usually about right.

Finally, (bonus) `'` are a pain with psql and prevent use - I normally  replace with a ``` but there might be a better way

Reply:
- controlling table name -> yep, but it's already mentioned in the article (line 13),
- batching -> added note
- `\`` -> if you mean in column titles, I also mentioned skipping the column names option in the article -> do you think that'll work?

Reply:
batching -> on line 36

Reply:
perfect